tests/src/subscribe/subscribe.test.ts

```diff
@@ -32,7 +37,7 @@ describe("test vault subscribe function", () => {
         expect(version).not.toBeNull();
     });
 
-    test("testPlaceOrder", async() => {
+    test.skip("testPlaceOrder", async() => {
```

Comment:
With now working PaymentService tests, there's no need to skip this test since it's a duplicate of paymentservice.test.ts#testPaceOrder. Just like "testGetOrder".

Reply:
Align with hive java.